tests/test_app/TestApp/Model/Behavior/Test4Behavior.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+namespace TestApp\Model\Behavior;
+
+use ArrayObject;
+use Cake\Event\EventInterface;
+use Cake\ORM\Behavior;
+use Cake\ORM\Query\SelectQuery;
+
+class Test4Behavior extends Behavior
```

Comment:
Class docblock missing. Short description of what the behavior does. Probably a (shortened) copy of the ``beforeFind()`` method doc block.